### DISCUSSION TITLE
HistoryGraph Bar Chart

### DISCUSSION THREAD

**Discussion by ahenric** [Zefau/ioBroker.jarvis#1960] [Q&A] [answered]
Wie kann man beim HistoryGraph Bar Chart die Breite der Balken verstellen, das sie z.B. nahtlos aneinander liegen? Macht man dies über den X-Axis Einstellungen?

Im Moment sehe ich immer nur dünne Balken, nur wenn man dann rein-zoomt werden sie breiter, aber auch dann liegen sie nicht aneinander.

**Comment by mcuiobroker** [comment, marked as answer]
Geht schon über die Series Einstellung:
![image](https://user-images.githubusercontent.com/84818934/195077975-49557f4a-6687-4948-9be7-0b4e199ebb73.png)
![image](https://user-images.githubusercontent.com/84818934/195078235-4301ccd9-285c-4b5e-b687-e37cb53eb7fb.png)

`{"barWidth": 20}`
![image](https://user-images.githubusercontent.com/84818934/195078103-5f77777e-88fa-4899-8530-d4d0b8f019fc.png)
